lsp-mode.el

```diff
@@ -2072,14 +2072,15 @@ The `:global' workspace is global one.")
 (lsp-defun lsp--headerline-breadcrumb-symbol-icon ((&DocumentSymbol :kind))
   "Build the SYMBOL icon for headerline breadcrumb."
   (when (require 'lsp-treemacs nil t)
-    (propertize " " 'display
-                (cl-list* 'image
-                          (plist-put
-                           (cl-copy-list
-                            (cl-rest (get-text-property
-                                      0 'display
-                                      (lsp-treemacs-symbol-icon kind))))
-                           :background (face-attribute 'header-line :background))))))
+    (concat (propertize " " 'display
+                        (cl-list* 'image
+                                  (plist-put
+                                   (cl-copy-list
+                                    (cl-rest (get-text-property
+                                              0 'display
+                                              (lsp-treemacs-symbol-icon kind))))
+                                   :background (face-attribute 'header-line :background))))
+            " ")))
```

Comment:
Do we really need this space?
I think in modeline/header line generally we only need prepend space for a component?

Reply:
The original treemacs icon is two spaces. When we recreate the icon, we should preserve that, e. g.

```
(length (treemacs-get-icon-value "c" nil lsp-treemacs-theme))
```